src/main/java/rs117/hd/model/objects/InheritTileColorType.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2021, 117 <https://twitter.com/117scape>
```

Comment:
This is quite picky, but it should probably have your name/github instead, or no copyright notice if you prefer that.

Reply:
I'll remove